mlir/lib/Dialect/XeGPU/Transforms/XeGPUWgToSgDistribute.cpp

```diff
@@ -29,6 +31,29 @@ using namespace mlir;
 
 namespace {
 
+static std::pair<SmallVector<int64_t>, int>
+getSgShapeAndCount(ArrayRef<int64_t> shape, xegpu::LayoutAttr layout) {
+  int count = 1;
+  SmallVector<int64_t> sgShape(shape);
+
+  if (layout && layout.isWgLayout()) {
```

Comment:
isWgLayout seems confusing, I think it should be called isSgLayout since it describes how the subgroups are laid out

Reply:
This is an interface defined in a previous PR. I think we can create a small fix PR if we plan to change it.